src/pages/about.astro

```diff
@@ -1,19 +1,29 @@
 ---
-import Layout from '../layouts/Layout.astro';
-import { getAbout } from '../services/contentful';
+import Layout from "../layouts/Layout.astro";
+import { getAbout } from "../services/contentful";
 import { documentToHtmlString } from "@contentful/rich-text-html-renderer";
 
-Astro.response.headers.set("Cache-Control", `max-age=${import.meta.env.CACHING_MAX_AGE}`)
+Astro.response.headers.set(
+  "Cache-Control",
+  `max-age=${import.meta.env.CACHING_MAX_AGE}`
+);
 
 // https://docs.astro.build/en/guides/cms/contentful/
 
 const about = await getAbout();
 ---
+
 <Layout title="DSSD Madison - About">
   <main>
-    <h1 class="text-2xl text-center p-4 text-gray-700">Who We Are</h1>
-    <article set:html={documentToHtmlString(about.whoWeAre)}></article>
-    <h1 class="text-2xl text-center p-4 text-gray-700">What We Do</h1>
-    <article set:html={documentToHtmlString(about.whatWeDo)}></article>
+    <div class="flex flex-col lg:flex-row">
+      <div class="p-10 m-10 bg-slate-200 shadow-xl rounded-xl">
```

Comment:
Ideally these blocks of repeated code would be components, like "AboutCard" or something. Don't worry about changing it though because we will probably eventually change the way this looks anyway when we decide on photos and designs to integrate into the site